### ISSUE TITLE
Does it demand from the root? 

### ISSUE THREAD

**Issue by AndreiGTA** [vfsfitvnm/frida-il2cpp-bridge#568]
Please tell me, does it require with root to connect via USB to Frida-il2cpp-bridge?

**Comment by bluewave41** [comment]
No, Frida in general doesn't require root. You do need to inject the gadget into an APK which is most easily done with something like https://github.com/sensepost/objection
